src/resolve/resolved-manager.c

```diff
@@ -1396,6 +1396,19 @@ void manager_flush_caches(Manager *m) {
         log_info("Flushed all caches.");
 }
 
+void manager_reexamine_servers(Manager *m) {
+        Iterator i;
+        Link *l;
+
+        dns_server_reexamine_all(m->dns_servers);
+        dns_server_reexamine_all(m->fallback_dns_servers);
+
+        HASHMAP_FOREACH(l, m->links, i)
+                dns_server_reexamine_all(l->dns_servers);
+
+        log_info("Re-examining all servers.");
```

Comment:
Yeah, this is misleading. "Re-examining" implies that there's some queries being sent out, while this just resets some stats.